doc/BUILD.md

```diff
@@ -120,21 +120,17 @@ Install the project in editable mode with its develop requirements:
     pip install -r requirements-test.txt
     pip install -e "."
 
-If you want to use PostgreSQL, MySQL or MariaDB database engine instead of
+If you want to use PostgreSQL, database engine instead of
 the default one (SQLite bundled with python), you need to install the python driver for those databases
 that are supported by SQLAlchemy.
```

Comment:
```suggestion
If you want to use PostgreSQL, database engine instead of the default one (SQLite bundled with 
python), you need to install the python driver for those databases that are supported by SQLAlchemy.
```